lib/elixir/lib/system.ex

```diff
@@ -42,6 +42,20 @@ defmodule System do
     end
   end
 
+  defmacrop get_revision do
```

Comment:
This is good. Let's drop the `:tag` from `build_info` and go only with `:revision`. :)